test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -114,6 +114,7 @@ describe('Transfer custom tokens @no-mmi', function () {
           smartContract,
         );
         await unlockWallet(driver);
+        console.log('FOO');
```

Comment:
I am sure you want to remove this :)

Reply:
😆 yes I definitely will! There are still a few things I need to clean up before officially taking it out of draft

Reply:
https://github.com/MetaMask/metamask-extension/pull/26952/commits/aeee661e73e9abd4021aedf01616ec39a07066e1